archinstall/lib/networking.py

```diff
@@ -1,4 +1,5 @@
 import fcntl
+import logging
 import os
 import logging
```

Comment:
We do have duplicate imports here tho, not sure this is needed. Same with `log` below.

Reply:
fixed